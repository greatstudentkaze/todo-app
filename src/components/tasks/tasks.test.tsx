import { shallow, ShallowWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';

import Tasks from './index';

describe('Tasks component', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Tasks />);
  });

  it('renders correctly', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
