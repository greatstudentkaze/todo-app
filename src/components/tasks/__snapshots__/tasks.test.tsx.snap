// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tasks component renders correctly 1`] = `
<section
  className="tasks"
>
  <h2
    className="tasks__title"
  >
    Список задач
  </h2>
  <Connect(TaskList) />
  <button
    className="button"
    onClick={[Function]}
    type="button"
  >
    Добавить
  </button>
</section>
`;
